[
  {
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "evaluationErrors": [],
    "probe": {
      "id": "<scrubbed>",
      "version": 0,
      "location": {
        "file": "/app/exception_replay_controller.py",
        "lines": [
          "61"
        ]
      }
    },
    "language": "python",
    "stack": [
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "exception_replay_multiframe",
        "lineNumber": 61
      },
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "captures": {
      "lines": {
        "61": {
          "arguments": {},
          "locals": {
            "@exception": {
              "type": "Exception",
              "fields": "<scrubbed>"
            }
          },
          "staticFields": "<scrubbed>",
          "throwable": {
            "type": "Exception",
            "message": "'multiple stack frames exception'",
            "stacktrace": [
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "exception_replay_multiframe",
                "lineNumber": 61
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_a",
                "lineNumber": 56
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_b",
                "lineNumber": 52
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_c",
                "lineNumber": 48
              },
              {
                "<runtime>": "<scrubbed>"
              }
            ]
          }
        }
      }
    },
    "duration": "<scrubbed>",
    "exception-id": "<scrubbed>"
  },
  {
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "evaluationErrors": [],
    "probe": {
      "id": "<scrubbed>",
      "version": 0,
      "location": {
        "file": "/app/exception_replay_controller.py",
        "lines": [
          "56"
        ]
      }
    },
    "language": "python",
    "stack": [
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "deep_function_a",
        "lineNumber": 56
      },
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "exception_replay_multiframe",
        "lineNumber": 61
      },
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "captures": {
      "lines": {
        "56": {
          "arguments": {},
          "locals": {
            "@exception": {
              "type": "Exception",
              "fields": "<scrubbed>"
            }
          },
          "staticFields": "<scrubbed>",
          "throwable": {
            "type": "Exception",
            "message": "'multiple stack frames exception'",
            "stacktrace": [
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "exception_replay_multiframe",
                "lineNumber": 61
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_a",
                "lineNumber": 56
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_b",
                "lineNumber": 52
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_c",
                "lineNumber": 48
              },
              {
                "<runtime>": "<scrubbed>"
              }
            ]
          }
        }
      }
    },
    "duration": "<scrubbed>",
    "exception-id": "<scrubbed>"
  },
  {
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "evaluationErrors": [],
    "probe": {
      "id": "<scrubbed>",
      "version": 0,
      "location": {
        "file": "/app/exception_replay_controller.py",
        "lines": [
          "52"
        ]
      }
    },
    "language": "python",
    "stack": [
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "deep_function_b",
        "lineNumber": 52
      },
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "deep_function_a",
        "lineNumber": 56
      },
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "exception_replay_multiframe",
        "lineNumber": 61
      },
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "captures": {
      "lines": {
        "52": {
          "arguments": {},
          "locals": {
            "@exception": {
              "type": "Exception",
              "fields": "<scrubbed>"
            }
          },
          "staticFields": "<scrubbed>",
          "throwable": {
            "type": "Exception",
            "message": "'multiple stack frames exception'",
            "stacktrace": [
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "exception_replay_multiframe",
                "lineNumber": 61
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_a",
                "lineNumber": 56
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_b",
                "lineNumber": 52
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_c",
                "lineNumber": 48
              },
              {
                "<runtime>": "<scrubbed>"
              }
            ]
          }
        }
      }
    },
    "duration": "<scrubbed>",
    "exception-id": "<scrubbed>"
  },
  {
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "evaluationErrors": [],
    "probe": {
      "id": "<scrubbed>",
      "version": 0,
      "location": {
        "file": "/app/exception_replay_controller.py",
        "lines": [
          "48"
        ]
      }
    },
    "language": "python",
    "stack": [
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "deep_function_c",
        "lineNumber": 48
      },
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "deep_function_b",
        "lineNumber": 52
      },
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "deep_function_a",
        "lineNumber": 56
      },
      {
        "fileName": "/app/exception_replay_controller.py",
        "function": "exception_replay_multiframe",
        "lineNumber": 61
      },
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "captures": {
      "lines": {
        "48": {
          "arguments": {},
          "locals": {
            "@exception": {
              "type": "Exception",
              "fields": "<scrubbed>"
            }
          },
          "staticFields": "<scrubbed>",
          "throwable": {
            "type": "Exception",
            "message": "'multiple stack frames exception'",
            "stacktrace": [
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "exception_replay_multiframe",
                "lineNumber": 61
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_a",
                "lineNumber": 56
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_b",
                "lineNumber": 52
              },
              {
                "fileName": "/app/exception_replay_controller.py",
                "function": "deep_function_c",
                "lineNumber": 48
              },
              {
                "<runtime>": "<scrubbed>"
              }
            ]
          }
        }
      }
    },
    "duration": "<scrubbed>",
    "exception-id": "<scrubbed>"
  }
]